source/renderer/app/components/staking/stake-pools/StakePoolTooltip.scss

```diff
@@ -197,7 +197,7 @@
 }
 
 .url {
-  @include link(--theme-staking-stake-pool-tooltip-link-color);
```

Comment:
I'd tune this variable to match transparency
but even now it looks good :+1:

Reply:
Let's leave it as is for now.